apps/network/lib/network.ex

```diff
@@ -1,18 +1,49 @@
-defmodule Network do
+defmodule Network.Application do
   @moduledoc """
-  Documentation for Network.
+  Documentation for Network Constructor.
   """
+  use Application
+  alias Station.QueryCollector, as: QueryCollector
```

Comment:
I just realized that the query collector is something to be created by api outside of network. Hence, it is appropriate to move the query collector to [util/lib](https://github.com/prasadtalasila/TransportScheduler/tree/ts-v1.0-alpha/apps/util/lib) and update the dependencies network module.
please remember to move the tests as well.